CHANGES.rst

```diff
@@ -776,6 +776,14 @@ API changes
     for parsing a file with blank lines in the header containing whitespace will
     need to be updated. [#2654]
 
+  - if any keys in the (formats, include_names, exclude_names) dictionary are
+    not to match any keys in the names dictionary then it's make warning to show
+    which keys in the (formats, include_names, exclude_names) dictionary are not
+    to match in names Dictionary. if any keys in the exclude_names dictionary
+    match to any keys in include_names dictionary then it's make warning to show
+    which keys in the exclude_names dictionary are match in include_names
+    dictionary. [#4508]
```

Comment:
This changelog entry will still need to be rewritten.  It really doesn't need to be so verbose, and it's also lacking context.    Something like 
```
Better validation of ``formats``, ``include_names``, and ``exclude_names`` arguments to ``ascii.write()``.
```
The details of this change really aren't deeply important from a user-perspective, and the issue reference allows someone to read up more on what is meant by this if they really care.